### DISCUSSION TITLE
Unable to install scikit-learn-intelex on Debian 12

### DISCUSSION THREAD

**Discussion by anjalirai-intel** [intel/scikit-learn-intelex#1545] [Q&A]
**Describe the bug**
With the PEP 668 changes, we are unable to install scikit-learn-intelex on host env, and I am not sure if this package is available by apt manager

**To Reproduce**
python3 -m pip install scikit-learn-intelex

**Expected behavior**
Is there a way we can install scikit-learn-intelex on the host system outside venv

**Output/Screenshots**
```
python3 -m pip install scikit-learn-intelex
error: externally-managed-environment

× This environment is externally managed
╰─> To install Python packages system-wide, try apt install
    python3-xyz, where xyz is the package you are trying to
    install.
    
    If you wish to install a non-Debian-packaged Python package,
    create a virtual environment using python3 -m venv path/to/venv.
    Then use path/to/venv/bin/python and path/to/venv/bin/pip. Make
    sure you have python3-full installed.
    
    If you wish to install a non-Debian packaged Python application,
    it may be easiest to use pipx install xyz, which will manage a
    virtual environment for you. Make sure you have pipx installed.
    
    See /usr/share/doc/python3.11/README.venv for more information.

note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
hint: See PEP 668 for the detailed specification.
```

**Environment:**
 - OS: Debian 12
 - Compiler: gcc (Debian 12.2.0-14) 12.2.0

**Comment by napetrov** [comment]
i think there is nothing specific to sklearnex package but just and overall enforcement with newer OSs

There is no native sklearnex package, so you can't follow this option. 
alternatives is using venv or pipx

Do you really need to install in host env?

**Comment by anjalirai-intel** [comment]
@napetrov I will try with pipx and update if it works or not

**Reply by anjalirai-intel** [reply]
Sorry for the delay, I tried with pipx, but it does not install all the dependencies even after adding --include-deps option

**Comment by syakov-intel** [comment]
Closing as per agreement with @anjalirai-intel
